app/routes.py

```diff
@@ -67,6 +67,12 @@ def create_movie_preference(user):
     return MoviePreferenceController().create(user)
 
 
+@app.route('/users/<user_id>/movie-preferences/<movie_preference_id>', methods=['PUT'])
```

Comment:
I think this should be a PATCH, PUT is a full replacement update I think

https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/PATCH

Reply:
huh TIL! I had always seen them used interchangeably